test-requirements.txt

```diff
@@ -3,4 +3,4 @@
 coverage
 flake8
 nose
-mock; python_version<='2.7'
+mock; python_version=='2.7'
```

Comment:
so version here could never include ".patch" (ie to be `2.7.5`)? if it could then would need to retain `<=`

Reply:
Not too sure. Shall I revert this to be on the safe side?